jobsapp/tests.py

```diff
@@ -1,3 +1,33 @@
 from django.test import TestCase
+import factory
+from . import models
+import pytest
+
+
+# List of factories
+class UserFactory(factory.Factory):
```

Comment:
I don't know how you want to do the test structure. Up to you or to the team @francescarpi but maybe you could create a `tests` folder in the project root. And inside of this folder you can add `jobsapp` and then inside of this one you can create test files. With this structure we will be able to copy only code inside the docker image if we need it.
Another thing are the factories. I would create a factory file like `factories.py` with the factories and then the test files could import this file.

Reply:
I approve your proposal, Jordi. It's ok for me create the `test` folder and put all factories in the `factories.py` file.